### DISCUSSION TITLE
Possibility of incremental write?

### DISCUSSION THREAD

**Discussion by sreerajchundayil** [nlohmann/json#3797] [Q&A]
I am populating an nlohmann::json object. Over 1 hour time it grows to 5gb in side. At the end I write it to a file.
I want to improve the performance of writing.
Is it possible to do a incremental write to the file?

**Comment by nlohmann** [comment]
No, this is currently not possible. I assume you write a large array - in that case, you could serialize the individual values and take care of commas and the brackets yourself.
